src/main/java/com/google/sps/servlets/NewRecipeServlet.java

```diff
@@ -27,9 +17,39 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // implement new recipe
+    String recipeName = request.getParameter("recipeName");
+    String tagsResponse = request.getParameter("tags");
+    String ingredientsResponse = request.getParameter("ingredients");
+    String stepsResponse = request.getParameter("steps");
+    String privacy = request.getParameter("privacy");
```

Comment:
It doesn't look like you use this anywhere, but I'm assuming that's later in the implementation?

Reply:
yup once we have public recipes